framework/src/executioners/FixedPointSolve.C

```diff
@@ -304,8 +317,21 @@ FixedPointSolve::solve()
 
```

Comment:
the line right above, should it be done by the convergence object?

Reply:
You mean `printFixedPointConvergenceHistory()`?

Reply:
This was related to your other question about moving the norm parameters - I'll move this too now.